File: src/app/api/models/booking-dto.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceDto } from './time-place-dto';
export interface BookingDto {
  airline?: null | string;
  arrival?: TimePlaceDto;
  departure?: TimePlaceDto;
  flightId?: string;
  numberOfSeats?: number;
  passengerEmail?: null | string;
  price?: null | string;
}

File: Entities/DTO/BookingDTO.cs
using Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.DTO
{
  public class BookingDTO
  {
    public Guid FlightId { get; set; }
    public string Airline { get; set; }

    public string Price { get; set; }
    public TimePlaceDTO Arrival { get; set; }
    public TimePlaceDTO Departure { get; set; }
    public string PassengerEmail { get; set; }
    public int NumberOfSeats { get; set; }


  }
}
